### PR TITLE
Ignore duplicate parent methods

### PR DESCRIPTION
**Pull request by EPashkin**
Part of https://github.com/gtk-rs/gir/issues/139
Applying https://github.com/gtk-rs/gir/pull/305.
Don't builds until https://github.com/gtk-rs/examples/pull/106 merged.

Not ready for merge as need "Update gir submodule"

### CODE REVIEW

File: src/auto/file_chooser_button.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// This file was generated by gir (e48471c) from gir-files (71d73f0)
+// This file was generated by gir (5be24b7+) from gir-files (71d73f0)
```
Comment (GuillaumeGomez) [inline src/auto/file_chooser_button.rs]: The hash is different here and ends with a '+'. Is this normal?

Comment (EPashkin) [inline src/auto/file_chooser_button.rs]: Yes, it just sample.

**Comment by EPashkin** [comment]
Ready for merge

**Comment by GuillaumeGomez** [comment]
Awesome! Thanks! (Waiting for appveyor before merging.)

**Pull request merged**